test/test_portsofcall.cpp

```diff
@@ -18,6 +18,8 @@
 #define CATCH_CONFIG_RUNNER
 #include "catch2/catch.hpp"
 
+#include "test_array.hpp"
```

Comment:
Not a fan of this... We should probably just copy the setup we have in singularity to do multiple test files in individual `cpp` files.